pycovid/pycovid.py

```diff
@@ -81,7 +92,7 @@ def getCovidCasesWide(countries=None, start_date=None, end_date=None, casetype=[
     iso_df = pd.read_csv('https://raw.githubusercontent.com/lukes/ISO-3166-Countries-with-Regional-Codes/master/slim-3/slim-3.csv')
     iso_df = iso_df[['name', 'alpha-3']]
     iso_df.loc[iso_df.name=="United States of America", 'name'] = 'US'
-    iso_df.loc[iso_df.name=="China", 'name'] = 'Mainland China'
+    iso_df.loc[iso_df.name=="China", 'name'] = 'China'
```

Comment:
Hum, it seems that this didn't solve the problem with getting the Chinese data. Did it?

Reply:
OK, it solved it for getCovidCasesWide, but not getCovidCases

Reply:
https://github.com/sudharshan-ashok/pycovid/pull/7